asset_stream_manager/main.cc

```diff
@@ -50,14 +50,6 @@ absl::Status Run(const AssetStreamConfig& cfg) {
       [&sm_server]() { return sm_server.Shutdown(); });
 
   RETURN_IF_ERROR(sm_server.Start(kSessionManagementPort));
-  if (!cfg.src_dir().empty()) {
```

Comment:
So you're taking out the dev startup mode? Would it be hard to continue supporting it? I always found it quite handy, especially for starting it with the debugger attached.

Reply:
Done. There are quite a few new flags we have to add, but it's possible.